packages/foundation/src/responsive/breakpoints-sizes.ts

```diff
@@ -8,5 +8,9 @@ export const BreakpointsSizes = {
   },
   l: {
     min: 992,
+    max: 1440,
+  },
+  xl: {
+    min: 1440,
```

Comment:
I noticed that in some bigger screens would be useful to have a bigger screen set up so I added one step bigger